core/bitarithm.c

```diff
@@ -31,10 +31,10 @@ unsigned bitarithm_msb(unsigned v)
     register unsigned shift;
 
     r =     (v > 0xFFFF) << 4; v >>= r;
-    shift = (v > 0xFF  ) << 3; v >>= shift; r |= shift;
-    shift = (v > 0xF   ) << 2; v >>= shift; r |= shift;
-    shift = (v > 0x3   ) << 1; v >>= shift; r |= shift;
-                                            r |= (v >> 1);
+    shift = (v > 0xFF) << 3; v >>= shift; r |= shift;
+    shift = (v > 0xF) << 2; v >>= shift; r |= shift;
+    shift = (v > 0x3) << 1; v >>= shift; r |= shift;
+    r |= (v >> 1);
```

Comment:
I remember that some people wanted the original formatting (I also think it looks quite nice, but it of course breaks any automation). Maybe surround it with `/* begin{code-style-ignore} */`/`/* end{code-style-ignore} */`?